libres/lib/enkf/local_obsdata.cpp

```diff
@@ -119,9 +119,10 @@ local_obsdata_get_node_active_list(local_obsdata_type *obsdata,
 }
 
 namespace {
-ActiveList &get_active_list(py::handle obj, const std::string &name) {
+const ActiveList *get_active_list(py::handle obj, const std::string &name) {
```

Comment:
The way this is supposed to work is that you create a `LocalObsDataNode` with a default `ActiveList` corresponding to all elements active, and then you can use the `get_active_list()` function to get a mutable `ActiveList` reference which can then be modified in a script. Right now when the returned `ActiveList` is enforced to be `const` the whole point of the `ActiveList` disappears. Of course that this PR was green is a clear sign that the testing was originally way under par.

Most observations are by nature localized in themselves, and there is really no need to split them in the way the `get_active_list()` allows for - the exception is seismic data which highly distributed spatially, and this type of split configuration make sense.